services/catalog/rs_server_catalog/user_catalog.py

```diff
@@ -1123,7 +1123,7 @@ async def dispatch(self, request, call_next):  # pylint: disable=too-many-branch
         response = await call_next(request)
 
         # Don't forward responses that fail
-        if response.status_code != 200:
+        if response.status_code not in [200, 201]:
```

Comment:
Could you use `if not response.is_success` instead ? @Padeanu what do you think ? 
```python
    @property
    def is_success(self) -> bool:
        """
        A property which is `True` for 2xx status codes, `False` otherwise.
        """
        return codes.is_success(self.status_code)
```

Reply:
Yes, i would be nice